redaxo/src/core/lib/packages/package.php

```diff
@@ -316,7 +316,9 @@ public function enlist()
 
         // add addon path for i18n
         if (is_readable($folder . 'lang')) {
```

Comment:
Ggf für separaten PR: super wäre wenn man den is_readable check wegbekommen würde.
(D.h. Wenn der i18n cache sich auch merken würde welche dateien es nicht gibt, bzw. Wenn man die annahme treffen könnte dass solange keine neuen addons aktiviert/deaktiviert/updated werden, dass man dann ja ohnehin alle lang dirs bereits kennt - voraussgesetzt man ist user und kein admin)

Reply:
Kann das is-redable hier jetzt weg?

Reply:
Falls du es überdehen hast...

Reply:
ja, bzw. mit der Zeile darunter tauschen, da vor `addDirectory` das dennoch geprüft werden sollte.